hack/cmd/README.md

```diff
@@ -0,0 +1,44 @@
+# login command
+
+## Procedure
+
+### Step 1. Install dex server to the kind cluster with kong
+
+```
+helm install dex -n auth deployments/dex-server --create-namespace
+```
+
+> [!NOTE]
+> see [llm-operator](https://github.com/llm-operator/llm-operator/tree/main/hack) for the kong installation.
+
+### Step 2. map issuer name
+
+Map the service name to the node IP (or localhost for Docker Desktop), and align it with the issuer URL.
+
+Example /etc/hosts setting for the Docker Desktop:
+
+```
+127.0.0.1 kong-kong-proxy.kong
```

Comment:
Let me check my understanding 

- The issuer URL is `http://kong-kong-proxy.kong/dex` so that the auth request is forwarded to Dex inside the k8s cluster.

- We need this setting in `/etc/hosts` since this login command attempts to access `http://kong-kong-proxy.kong/dex` outside from the k8s cluster.

Reply:
I think we can remove this by adding the following Go code:

```go
			dialer := &net.Dialer{}
			http.DefaultTransport.(*http.Transport).DialContext = func(ctx context.Context, network, addr string) (net.Conn, error) {
				if addr == "kong-kong-proxy.kong:80" {
					addr = "127.0.0.1:80"
				}
				return dialer.DialContext(ctx, network, addr)
			}

```

Reply:
Also need to replace `u, err := url.Parse(path.Join(issuerURL, "auth"))`  to use local addr.

Reply:
Sorry for the lack of explanation; I'll add a comment. 
As a background, dex reject authentication If the request destination does not match the issue URL (e.g., 127.0.0.1 != kong-kong-proxy.kong). So, we need to open dex issuer URL in a browser. I added the host's setting as an easy way to access the dex inside and outside the k8s cluster with the same hostname.

Reply:
Thanks! Understood. 

As updating /etc/hosts is tedious, it would be nice if the CLI command has an option to handle the address override like the above code snippet.